test/kafka_test.go

```diff
@@ -87,16 +87,13 @@ func (s *CLITestSuite) TestKafka() {
 		{args: "kafka acl list --cluster lkc-acls", fixture: "kafka/acl/list-cloud.golden"},
 		{args: "kafka acl list --cluster lkc-acls -o json", fixture: "kafka/acl/list-json-cloud.golden"},
 		{args: "kafka acl list --cluster lkc-acls -o yaml", fixture: "kafka/acl/list-yaml-cloud.golden"},
-		{args: "kafka acl list --principal User:12345", fixture: "kafka/acl/err-numeric-id.golden", exitCode: 1},
```

Comment:
is `principal` no longer supported?

Reply:
It is, but principals containing numeric IDs are not.

Reply:
would be good to still have a few `principal` tests?

Reply:
We still do!